### DISCUSSION TITLE
[x]it! for ecode

### DISCUSSION THREAD

**Discussion by SpartanJ** [jotaen/xit#61] [Show & tell]
Hi!
I'm the author of [ecode](https://github.com/SpartanJ/ecode) an open-source code editor. ecode does support [x]it! and I'm wondering if it would be possible to add it to the Editor Support list on the [site](https://xit.jotaen.net/).
Thanks!

**Comment by jotaen** [comment]
Terrific, thanks for making this. I’ve linked it 🎉

**Reply by SpartanJ** [reply]
Awesome! Thank you very much!
